diff --git a/dev-support/ranger-docker/.env b/dev-support/ranger-docker/.env
index e3dc419..d774ee9 100644
--- a/dev-support/ranger-docker/.env
+++ b/dev-support/ranger-docker/.env
@@ -2,8 +2,8 @@ BUILD_HOST_SRC=true
 SKIPTESTS=true
 GIT_URL=https://github.com/apache/ranger.git
 BRANCH=ranger-2.6
-PROFILE=
-BUILD_OPTS=
+PROFILE=ranger-tagsync
+BUILD_OPTS=-Denunciate.skip=true
 
 # To build Trino plugins package, use following PROFILE
 # PROFILE=ranger-jdk11,!all,!linux
diff --git a/pom.xml b/pom.xml
index 344dadb..1fe78a0 100644
--- a/pom.xml
+++ b/pom.xml
@@ -885,6 +885,12 @@
                 <module>ugsync-util</module>
             </modules>
         </profile>
+        <profile>
+            <id>ranger-tagsync</id>
+            <modules>
+                <module>tagsync</module>
+            </modules>
+        </profile>
         <profile>
             <id>ranger-hdfs-plugin</id>
             <modules>
